OONIProbeUnitTests/TestKeysTest.m

```diff
@@ -0,0 +1,249 @@
+    testKeys.advanced = [Advanced new];
+    XCTAssert([[testKeys getMaxPing] isEqualToString:NOT_AVAILABLE]);
+    testKeys.advanced.max_rtt = [NSNumber numberWithDouble:0];
+    XCTAssert([[testKeys getMaxPing] isEqualToString:@"0.0"]);
+}
+
+- (void)testMss {
+    XCTAssert([[testKeys getMSS] isEqualToString:NOT_AVAILABLE]);
+    testKeys.advanced = [Advanced new];
+    XCTAssert([[testKeys getMSS] isEqualToString:NOT_AVAILABLE]);
+    testKeys.advanced.mss = [NSNumber numberWithDouble:0];
+    XCTAssert([[testKeys getMSS] isEqualToString:@"0"]);
+}
+
+- (void)testTimeouts {
+    XCTAssert([[testKeys getTimeouts] isEqualToString:NOT_AVAILABLE]);
+    testKeys.advanced = [Advanced new];
+    XCTAssert([[testKeys getTimeouts] isEqualToString:NOT_AVAILABLE]);
+    testKeys.advanced.timeouts = [NSNumber numberWithDouble:0];
+    XCTAssert([[testKeys getTimeouts] isEqualToString:@"0"]);
+}
+
+- (void)testMedianBitrate {
+    XCTAssert([[testKeys getMedianBitrate] isEqualToString:NOT_AVAILABLE]);
+    testKeys.simple = [Simple new];
+    XCTAssert([[testKeys getMedianBitrate] isEqualToString:NOT_AVAILABLE]);
+    testKeys.simple.median_bitrate = [NSNumber numberWithDouble:0];
+    XCTAssert([[testKeys getMedianBitrate] isEqualToString:@"0.00"]);
+    testKeys.simple.median_bitrate = [NSNumber numberWithDouble:10];
+    XCTAssert([[testKeys getMedianBitrate] isEqualToString:@"10.0"]);
+}
+
+- (void)testMedianBitrateUnit {
+    XCTAssert([[testKeys getMedianBitrateUnit] isEqualToString:NOT_AVAILABLE]);
+    testKeys.simple = [Simple new];
+    XCTAssert([[testKeys getMedianBitrateUnit] isEqualToString:NOT_AVAILABLE]);
+    testKeys.simple.median_bitrate = [NSNumber numberWithDouble:0];
+    XCTAssert([[testKeys getMedianBitrateUnit] isEqualToString:NSLocalizedString(@"TestResults.Kbps", nil)]);
+    testKeys.simple.median_bitrate = [NSNumber numberWithDouble:1000];
+    XCTAssert([[testKeys getMedianBitrateUnit] isEqualToString:NSLocalizedString(@"TestResults.Mbps", nil)]);
+    testKeys.simple.median_bitrate = [NSNumber numberWithDouble:1000*1000];
+    XCTAssert([[testKeys getMedianBitrateUnit] isEqualToString:NSLocalizedString(@"TestResults.Gbps", nil)]);
+}
+
+- (void)testVideoQuality {
+    XCTAssert([[testKeys getVideoQuality:false] isEqualToString:NOT_AVAILABLE]);
+    testKeys.simple = [Simple new];
+    XCTAssert([[testKeys getVideoQuality:false] isEqualToString:NOT_AVAILABLE]);
+    testKeys.simple.median_bitrate = [NSNumber numberWithDouble:0];
+    XCTAssert([[testKeys getVideoQuality:false] isEqualToString:@"240p"]);
+}
+
+- (void)testPlayoutDelay {
+    XCTAssert([[testKeys getPlayoutDelay] isEqualToString:NOT_AVAILABLE]);
+    testKeys.simple = [Simple new];
+    XCTAssert([[testKeys getPlayoutDelay] isEqualToString:NOT_AVAILABLE]);
+    testKeys.simple.min_playout_delay = [NSNumber numberWithDouble:0];
+    XCTAssert([[testKeys getPlayoutDelay] isEqualToString:@"0.00"]);
+}
```

Comment:
This test is great and I think it's useful because it's checking that our custom attributes of testKeys are working as expected.
It is also a little bit tautological, but it's a great start.